### PR TITLE
Url config + fixes for the CI in Jenkins-X

### PR DESCRIPTION
**Pull request by Rude-Monkey**
tumblr_p6d7x1rq691qjztgpo1_400

### CODE REVIEW

**Comment by Rude-Monkey** [comment]
:star: PR built and available in a preview environment **aya-experience-aya-day-weather-app-pr-5** [here](http://aya-day-weather-app.jx-aya-experience-aya-day-weather-app-pr-5.kube.gilleslabs.xyz)

**Comment by Rude-Monkey** [comment]
:star: PR built and available in a preview environment **aya-experience-aya-day-weather-app-pr-5** [here](http://aya-day-weather-app.jx-aya-experience-aya-day-weather-app-pr-5.kube.gilleslabs.xyz)

File: nuxt.config.js [hunk 1]
```diff
@@ -1,3 +1,13 @@
+let serviceUrl;
+
+if(process.env.NODE_ENV === 'container'){
```
Comment (FBerthelot) [inline nuxt.config.js]: https://www.npmjs.com/package/config :)

File: package.json [hunk 2]
```diff
@@ -14,7 +14,10 @@
     "test": "npm run unit && npm run integration",
     "unit": "jest --ci --runInBand test/unit/*.test.js",
     "integration": "jest --ci --runInBand --forceExit test/integration/*.test.js",
-    "tdd": "jest --watch test/**/*.test.js"
+    "tdd": "jest --watch test/**/*.test.js",
+    "jenkins-x-test": "NODE_ENV=jenkinsx nuxt build",
```
Comment (FBerthelot) [inline package.json]: 💩

**Comment by Rude-Monkey** [comment]
:star: PR built and available in a preview environment **aya-experience-aya-day-weather-app-pr-5** [here](http://aya-day-weather-app.jx-aya-experience-aya-day-weather-app-pr-5.kube.gilleslabs.xyz)

**Comment by Rude-Monkey** [comment]
:star: PR built and available in a preview environment **aya-experience-aya-day-weather-app-pr-5** [here](http://aya-day-weather-app.jx-aya-experience-aya-day-weather-app-pr-5.kube.gilleslabs.xyz)

**Comment by Rude-Monkey** [comment]
:star: PR built and available in a preview environment **aya-experience-aya-day-weather-app-pr-5** [here](http://aya-day-weather-app.jx-aya-experience-aya-day-weather-app-pr-5.kube.gilleslabs.xyz)

**Comment by Rude-Monkey** [comment]
:star: PR built and available in a preview environment **aya-experience-aya-day-weather-app-pr-5** [here](http://aya-day-weather-app.jx-aya-experience-aya-day-weather-app-pr-5.kube.gilleslabs.xyz)

**Comment by Rude-Monkey** [comment]
:star: PR built and available in a preview environment **aya-experience-aya-day-weather-app-pr-5** [here](http://aya-day-weather-app.jx-aya-experience-aya-day-weather-app-pr-5.kube.gilleslabs.xyz)

**Pull request merged**